storage/src/main/java/tech/pegasys/teku/storage/server/kvstore/BlindedBlockKvStoreDatabase.java

```diff
@@ -262,6 +260,13 @@ public Optional<Bytes32> getFinalizedBlockRootBySlot(final UInt64 slot) {
     return dao.getFinalizedBlockRootAtSlot(slot);
   }
 
+  @Override
+  public Optional<ExecutionPayload> getExecutionPayload(
+      final Bytes32 blockRoot, final UInt64 slot) {
+    final Optional<Bytes> maybePayload = dao.getExecutionPayload(blockRoot);
```

Comment:
Was there a reason the dao is treating the payload as raw bytes instead of the usual approach where the column is typed and then we use getRaw if we need the raw bytes?

Reply:
it likely needs a deserializer written, has to have the spec to get the payload decoded currently...

Reply:
We'd have the same requirement for blocks so that should work out ok.  But worth leaving to a follow up PR.

Reply:
blocks deserializer can read from the body, we wont be able to... but yes, we can investigate

Reply:
ah yes interesting point.  That could well be the reason we did it this way....